managed/models/percona_sso_model_helpers.go

```diff
@@ -23,25 +23,18 @@ import (
 	"io"
 	"net/http"
 	"net/url"
-	"sync"
 	"time"
 
 	"github.com/pkg/errors"
 	"gopkg.in/reform.v1"
 )
 
-var (
-	perconaSSOMtx           sync.Mutex
-	ErrNotConnectedToPortal = errors.New("PMM Server is not connected to Portal")
-)
+var ErrNotConnectedToPortal = errors.New("PMM Server is not connected to Portal")
 
 // GetPerconaSSODetails returns PerconaSSODetails if there are any, error otherwise.
 // Access token is automatically refreshed if it is expired.
 // Get, check eventually refresh done in one tx.
 func GetPerconaSSODetails(ctx context.Context, q *reform.Querier) (*PerconaSSODetails, error) {
-	perconaSSOMtx.Lock()
```

Comment:
removed this mutex since it was not guarding anything in memory

Reply:
@oksana-grishchenko could you confirm this? i don't see any concurrent access here? why was this mutex added do you remember?

Reply:
During the refresh token process the the AccessToken that stored in DB [got updated](https://github.com/percona/pmm/blob/3df82544be0ab59b259c3df2fac5ba71f3c20d0d/managed/models/percona_sso_model_helpers.go#L106). I suppose this was why this mutex were there

Reply:
i still don't understand if the write is happening on disk then why we need a mutex, but i will keep it for now just so that this doesn't break anything. thanks.